torchgen/executorch/api/unboxing.py

```diff
@@ -0,0 +1,224 @@
+from dataclasses import dataclass
+from typing import Callable, List, Sequence, Tuple, Union
+
+from mypy_extensions import DefaultNamedArg, NamedArg
+
+from torchgen.api.types import Binding, CType, NamedCType, SpecialArgName
+from torchgen.model import (
+    Argument,
+    BaseTy,
+    BaseType,
+    ListType,
+    NativeFunction,
+    OptionalType,
+    Type,
+)
+
+connector = "\n\t"
```

Comment:
you sure you want to do this with tabs :think:

Reply:
it doesn't really matter lol. what's the canonical way of doing this? whitespaces?

Reply:
well, it's pretty tough to get f-strings to indent correctly, but yeah I usually just do spaces